ucsb-cs156-f23/team03-f23-5pm-4
# Create Page for UCSBDiningCommonsMenuItem plus tests and stories

Create Page for UCSBDiningCommonsMenuItem, with tests and stories.

<img width="1727" alt="Screenshot 2023-11-07 at 2 17 54 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-5pm-4/assets/5925080/23254b29-1548-4ebc-87b6-d28a9978d095">

[Storybook](https://ucsb-cs156-f23.github.io/team03-f23-5pm-4/prs/75/storybook/?path=/story/pages-ucsbdiningcommonsmenuitem-ucsbdiningcommonsmenuitemcreatepage--default)
Testing instructions:
* log into [dokku deployment](https://team03-ajs808-dev.dokku-04.cs.ucsb.edu/)
* navigate to [create page](https://team03-ajs808-dev.dokku-04.cs.ucsb.edu/diningcommonsmenuitem/create)
* input fields and press create
* navigate to [swagger](https://team03-ajs808-dev.dokku-04.cs.ucsb.edu/swagger-ui/index.html#/UCSBDiningCommonsMenuItems/allUCSBDiningCommonMenuItems)
* verify that GET `all` endpoint includes the item you just created
